### PR TITLE
[RDAPI-28853] [RDAPI-28556] Body serialization important changes

### PR DESCRIPTION
**Pull request by JohnD-Axway**
http://pr-2570.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_relnotes/20230228_apimgr_relnotes/#serialization-of-xml-payload

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 33b7f86 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2570.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md [hunk 1]
```diff
@@ -279,6 +287,10 @@ name=felix&age=5&time=2022-12-31T22:59:01Z
 
 Related Issue: RDAPI-29098
 
+### Enable Unsafe Legacy Renegotiation on TLS Listeners
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md:290]: @JohnD-Axway what's the related Jira ticket ?

File: content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md [hunk 2]
```diff
@@ -101,6 +101,14 @@ API Gateway SDK libraries have been updated as follows:
 | `com/vordel/mime/RawBody` | Javadoc enhancements |
 | `com/vordel/mime/XMLBody` | Javadoc enhancements |
 
+### Serialization of XML Payload
+
+XML Bodies will only be serialized when the XML Body has been modified. Prior to this XML Bodies would be re-serialized when they were parsed, this would result in header changes. A new system property "com.axway.data.serialize.xmlbody.always" has been defined to allow you to switch back to the old behavior. Default value is false. To enable the re-serialization of an XML body, add `<SystemProperty name="com.axway.data.serialize.xmlbody.always" value="true" />` to your jvm.xml file.
+
+### XMLBody, JSONBody and RawBody source is now private
+
+Body classes have changed in this release to prevent double buffering of the body source. The contract of these classes have changed. Scripts that use these classes should be reviewed. Please refer to SDK Library updates for more information.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md:110]: @JohnD-Axway do customers know where to find the SDK Library updates?

Comment (JohnD-Axway) [reply content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md]: [RDAPI-29275](https://jira.axway.com/browse/RDAPI-29275)

Comment (JohnD-Axway) [reply content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md]: No, I just followed up on this an apparently we don't publish them anymore so this statement will be removed

File: content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md (hunk 1 above)
Comment (andreamussap) [reply content/en/docs/apim_relnotes/20230228_apimgr_relnotes.md:290]: RDAPI-28853

**Review by mbarova — APPROVED** [review]

**Pull request merged**